app/Http/Controllers/DashboardController.php

```diff
@@ -40,6 +41,34 @@ public function getExpenses()
     public function getExpenseSummary($expenses){
       //make an summary
       //return new
+        $summary = array();
+        $owed = DB::table('expenses AS e')
+            ->leftjoin('sharedexpenses AS se', 'e.id', '=', 'se.expense_id')
+            ->select('e.*', 'se.secondary_username', 'se.amount_owed')
+            ->where('e.owner_username', Auth::user()->username)
+            //->get();
+            ->sum('se.amount_owed');
+        $owing = DB::table('expenses AS e')
+            ->leftjoin('sharedexpenses AS se', 'e.id', '=', 'se.expense_id')
+            ->select('e.*', 'se.secondary_username', 'se.amount_owed')
+            ->where('se.secondary_username', '=', Auth::user()->username)
+            //->get();
+            ->sum('se.amount_owed');
+        // ttl means amount spent
+            // assuming that means
+        $ttl = DB::table('expenses AS e')
+            ->where('e.owner_username', '=', Auth::user()->username)
```

Comment:
I actually think we should remove total from the display as we have no way of "settling" personal expenses, and this very quick will be a complicated query.

Or unless we're under the idea that TTL is how much you spent personally, then which we'd have to actually do a NOT IN query to remove the expenses that are shared.